.github/workflows/notify_slack.yml

```diff
@@ -0,0 +1,39 @@
+name: Test all warehouse platforms
+on:
+  workflow_call:
+    inputs:
+      result:
+        type: string
+        required: true
+      run_id:
+        type: string
+        required: false
+      workflow_name:
+        type: string
+        required: true
+
+jobs:
+  notify:
+    name: Notify Slack Failure
+    runs-on: ubuntu-latest
+    if: ${{ inputs.result == 'failure' }}
```

Comment:
The caller should choose when to call this job.
If someone called `Notify Slack` it should run.

Reply:
I'm coloring the message, so if we'll want to add a "success" message we'll add another job with a different if statement. For now doesn't feel like a big deal in the implementation

Reply:
If you're not going to change it, then I think we should rename this workflow and step to `notify_slack_failure` and remove the `if`.

Reply:
But I might add a success job. thats what I'm saying